NetCoreRestApi/DataLayer.EF/Entities/ProductEntity.cs

```diff
@@ -3,9 +3,9 @@
 namespace DataLayer.EF.Entities
 {
     public class ProductEntity : ItemEntity<int>
-    {      
-        public ICollection<CategoryEntity> Categories { get; set; }
+    {
         public decimal Price { get; set; }
         public int AvailableCount { get; set; }
+        public IEnumerable<CategoryEntity> Categories { get; set; }
```

Comment:
Revert this change because this is entity so it should be ICollection

Reply:
fixed